master/internal/rm/kubernetesrm/jobs.go

```diff
@@ -545,6 +545,12 @@ func (j *jobsService) VerifyNamespaceExists(namespaceName string) error {
 	return j.verifyNamespaceExists(namespaceName)
 }
 
+func (j *jobsService) RemoveEmptyNamespace(namespaceName string, clusterName string) error {
```

Comment:
it would also be great to see a new test added for this func

Reply:
done